src/components/common/view/WavesBackground.native.js

```diff
@@ -17,9 +17,8 @@ const styles = StyleSheet.create({
   wavesBackground: {
     position: 'absolute',
     bottom: 0,
-    left: -15,
     width: '310%',
-    height: '100%',
+    height: '110%',
```

Comment:
that's strange, why is that?
does this work both on web and mobile?

Reply:
I really don't know why 100% is not enough, and also I really don't know why width was 310%. But by some reason with 100% height svg waves is not fully covered the screen. It's change only for native because I changed .native file. For web nothing changed.

Reply:
@Zhigamovsky may be the issue inside svg file itself ? there's an [https://developer.mozilla.org/ru/docs/Web/SVG/Attribute/viewBox](viewBox) property could adjust svg drawing relative to the canvas area. Probably it just set incorrectly so part of the drawing is cutted

Reply:
No, I think have another reason. Look https://stackoverflow.com/questions/19484707/how-can-i-make-an-svg-scale-with-its-parent-container. We can scale up by width (scale to parent width), or by height( scale to parent height). But not the both (with saving proportionals). We can scale up by height and width without saving proportionals with prop preserveAspectRatio="none", but it is not what we need. In web (not a react native web) we can wrap the svg in div with width="100%" and height="auto" and svg will scale up to parent dimensions with saving proportions. But in RN we cannot do it (we have not "auto" value for height). So I just calucalted aspectRatio value for svg wrapper based on height and width that svg. Svg fully cover the screen.